src/ObjectPool/src/ObjectPool.cs

```diff
@@ -3,15 +3,31 @@
 
 namespace Microsoft.Extensions.ObjectPool
 {
+    /// <summary>
+    /// A pool of objects.
+    /// </summary>
+    /// <typeparam name="T">The type of objects to pool.</typeparam>
     public abstract class ObjectPool<T> where T : class
     {
+        /// <summary>
+        /// Returns an object from the pool.
```

Comment:
`Gets an object from the pool.` It's confusing to use the word `Return` in the `Get` method when there's also a `Return` method!

Reply:
Also, I think this isn't even quite true. All this does is guarantee you'll get an object of the desired type. If there are any available in the pool, you get one from the pool. If the pool is empty, you get a new one. I think it's crucial to describe this.